doc/api/readline.md

```diff
@@ -904,7 +914,8 @@ setTimeout(() => ac.abort(), 10000);
 
 If this method is invoked as it's util.promisify()ed version, it returns a
 Promise that fulfills with the answer. If the question is canceled using
-an `AbortController` it will reject with an `AbortError`.
+an `AbortController` it will reject with an `AbortError`. If the question is
+called after `rl.close()`, it returns a promise in a never settling state.
```

Comment:
I'm not sure what's the correct wording here (https://tc39.es/ecma262/#sec-promise-objects) 🤔
```suggestion
called after `rl.close()`, it returns a forever-pending promise.
```
```suggestion
called after `rl.close()`, it returns a promise that will never settle.
```
```suggestion
called after `rl.close()`, it returns an unresolved promise.
```